src/indicators/drought/chirpsgefs_rainfallforecast.py

```diff
@@ -0,0 +1,227 @@
+"""
+Download CHIRPSGEFS raster data and extract statistics per admin region
+
+[CHIRPS-GEFS](https://chc.ucsb.edu/data/chirps-gefs) is the bias-corrected version of GEFS.
+GEFS is the Global Ensemble Forecast System from NOAA.
+CHIRPS observational data is used for the bias-correction.
+The forecast is published each day for the whole world with a 0.05 resolution.
+It is relatively new, started in 2018, and while it seems a respected source, little research articles exist around it.
+However, GEFS is a well-established source. Forecasts are available starting from 2000
+
+Data limitations:
+- No CHIRPS-GEFS data is available from 01-01-2020 till 05-10-2020.
+This data is available from the [older version of the model](https://data.chc.ucsb.edu/products/EWX/data/forecasts/CHIRPS-GEFS_precip/15day/Africa/precip_mean/),
+but our contact at CHC recommended to not use this
+
+Assumptions
+- The grid cell size is small enough to only look at cells with their centre within the region, not those touching
+"""
+
+import pandas as pd
```

Comment:
Small PEP-8 comment: standard library imports should come before third party ones, see [here](url)

Reply:
A part of the link seems to be missing :(  https://github.com/OCHA-DAP/pa-anticipatory-action/pull/url

Reply:
nvm, now understood what you meant after reading [the documentation](https://www.python.org/dev/peps/pep-0008/#imports), thanks!

Reply:
Oops yes that is the link I meant to add, sorry about that ;)